gpu-pytorch/environment.yml

```diff
@@ -113,13 +113,13 @@ dependencies:
  - python-flatbuffers  # for onxxruntime-gpu
  - cupy
  - efficientnet-pytorch
- - hvplot==0.7.0  # https://github.com/holoviz/hvplot/issues/576
```

Comment:
This pin was for the plots below https://planetarycomputer.microsoft.com/docs/tutorials/landcover/#Predicting-land-cover-for-each-pixel. I haven't investigated if there's an alternative way to specify the colormap that doesn't hit that.